recipes/cyipopt/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set name = "cyipopt" %}
+{% set version = "0.1.7" %}
+{% set sha256 = "e62d4265743078c60f0af1426218310a9fbc1b6d0d0e60879e172d569dd3343d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://github.com/matthias-k/cyipopt/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # TODO: Once ipopt is available for Windows in conda forge, remove the
+  # following line.
+  skip: True  # [win]
+  # NOTE : Set CC to force gcc on OSX.
+  script: CC=gcc python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - toolchain
+    - gcc
+    - pkg-config  # [osx]
+    - python
+    - ipopt
+    - setuptools
+    - cython
+    - numpy  # [not py36]
+    # NOTE : During the build stage NumPy 1.9.3 is downloaded instead of the
```

Comment:
If you look at the earlier ciricleci builds on my first commits, numpy 1.9.3 is downloaded instead of the latest version in the build section. I couldn't figure out why it does this.

And yes, the extension uses the numpy ABI, I think. It is all Cython code with a cimport numpy. I can pin to the latest version of NumPy.